server/views/pages/overview/partials/overviewTabContents.njk

```diff
@@ -1,28 +1,31 @@
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
-
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    {% set insetHtml %}
-      <div>
-        <strong>Goals</strong>
+    <div class="moj-page-header-actions">
```

Comment:
How does this change in markup render?

Reply:
This uses the [MoJ header actions component](https://design-patterns.service.justice.gov.uk/components/page-header-actions/), using an inset text wasn't appropriate here so it displays a button next to a heading level. The designs have also been updated in the prototype https://hmpps-education-and-work-plan-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/v5/dashboard-pre-induction though the markup is wrong.